src/context/filter/filterContext.js

```diff
@@ -0,0 +1,17 @@
+import { createContext, useContext } from 'react';
+import { useReducer } from 'react';
+import { filterReducer, initialState } from '../../reducer/filterReducer';
+const FilterContext = createContext();
```

Comment:
You could add a default value to the `FilterContext` here, as in case you miss wrapping the component in context then it would act as a fallback and won't give unnecessary errors